docs/dev-environments/besu/cleanup.sh

```diff
@@ -13,11 +13,57 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+check_and_exit()
+{
+    if [ $? -ne 0 ]
+    then
+        echo "Script failed."
+        exit 1
+    else
+        echo "Done."
+    fi
+}
 
+echo ""
+echo "====================================================="
+echo "STEP 1 :: Stop HL Besu network and destroy containers"
+echo "====================================================="
+docker-compose down
+check_and_exit
+
+echo ""
+echo "====================================================="
+echo "STEP 2 :: Stop truffle setup container and destroy"
+echo "====================================================="
+docker-compose -f docker-compose-truffle.yaml down
+check_and_exit
+
+echo ""
+echo "====================================================="
+echo "STEP 3 :: Delete persistent blockchain data"
+echo "====================================================="
 # To clean up all persistent data for a running node
 cd ./besu/node1
 sudo rm -rf caches database DATABASE_METADATA.json
-
+check_and_exit
```

Comment:
You could do something similar to error_exit() in https://github.com/hyperledger/avalon/blob/master/tools/rebuild.sh

for eg: `sudo rm -rf caches database DATABASE_METADATA.json || error_exit "Failed to delete data in besu node1"`
```
function error_exit()
{
    echo "[Error]: " ${*}
    exit 1
}
```